src/Validation/Rule/SupportedRules.php

```diff
@@ -32,16 +27,15 @@ class SupportedRules
     /**
      * @return array
      */
-    public static function get(): array
+    public static function getNew(): array
```

Comment:
Not sure if the method name really needs to change

Reply:
Maybe, I did it for clarity.